extendedset/src/main/java/org/apache/druid/extendedset/intset/AbstractIntSet.java

```diff
@@ -1,19 +1,20 @@
-/* 
- * (c) 2010 Alessandro Colantonio
- * <mailto:colanton@mat.uniroma3.it>
- * <http://ricerca.mat.uniroma3.it/users/colanton>
- *  
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one

```

Comment:
Based on [this discussion](https://github.com/apache/incubator-druid/pull/6215#discussion_r212155496) I don't think this license header should be changed.

Reply:
@clintropolis you're probably right, will revert